src/Chapter2.hs

```diff
@@ -501,8 +513,9 @@ True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
-
+isThird42 :: (Eq a, Num a) => [a] -> Bool
+isThird42 (_:_:subject:_) = subject == 42
```

Comment:
You don't need to compare `subject` with 42 separately, you can pattern match on `42` directly in the pattern.
```suggestion
isThird42 (_:_:42:_) = True
```

Reply:
O.O totally forgot about that, thanks